src/presentational-components/myUserAccess/bundles.js

```diff
@@ -27,14 +32,16 @@ export const bundleData = [
       compliance: '/compliance',
       drift: '/drift',
       'image builder': '/image-builder',
-      patch: '/patch',
-      vulnerability: '/vulnerability',
+      malware detection: '/malware-detection',
```

Comment:
maybe there is missing "_" between `malware` and `detection`?
```suggestion
      malware_detection: '/malware-detection',
```